subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/TaskUpToDateIntegrationTest.groovy

```diff
@@ -269,4 +269,35 @@ class TaskUpToDateIntegrationTest extends AbstractIntegrationSpec {
         then:
         executedAndNotSkipped ":customTask"
     }
+
+    def "changing default excludes leaves task up-to-date"() {
```

Comment:
The test name suggests it's the default excludes you are changing, I think `"changes to inputs that are excluded by default leave task up-to-date"` is a better name.

Reply:
Changed.